doc/source/index.md

```diff
@@ -0,0 +1,328 @@
+---
+html_theme.sidebar_secondary.remove:
+sd_hide_title: true
+---
+
+<!-- CSS overrides on the homepage only -->
```

Comment:
could we also set width to 70rem here and have it be only for the homepage? I like the homepage a bit wider if there is enough space, especially for the hero element.